unit-tests/live/options/test-drops-on-set.py

```diff
@@ -2,7 +2,6 @@
 # Copyright(c) 2020 Intel Corporation. All Rights Reserved.
 
 #test:device L500*
-#test:device D400*
```

Comment:
@remibettan I don't understand... if we removed D400, why did we add code to `rspy.test` specific to D400? Is this temporary?